docs/tutorials/traefik-proxy.md

```diff
@@ -0,0 +1,98 @@
+# Configuring ExternalDNS to use the Traefik Proxy Source
+
+This tutorial describes how to configure ExternalDNS to use the Traefik Proxy source.
+It is meant to supplement the other provider-specific setup tutorials.
+
+## Manifest (for clusters without RBAC enabled)
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: external-dns
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+      - name: external-dns
+        # update this to the desired external-dns version
+        image: k8s.gcr.io/external-dns/external-dns:v0.12.2
+        args:
+        - --source=traefik-proxy
+        - --provider=aws
+        - --registry=txt
+        - --txt-owner-id=my-identifier
+```
+
+## Manifest (for clusters with RBAC enabled)
+
+Could be change if you have mulitple sources
```

Comment:
I am not sure what this sentence mean and it contains a typo. Can you please correct it?

Reply:
That is a fair point. I referenced the [gloo proxy tutorial](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/gloo-proxy.md?plain=1#L35) and carried it over.

Would you like me to fix/remove it from them as well?